__tests__/kata5.reachDestination.test.js

```diff
@@ -2,6 +2,7 @@ const { reachDestination } = require('../src');
 
 describe('reachDestination', () => {
   it('returns string with estimated time of arrival', () => {
-    
+    expect(reachDestination(44,10)).toEqual(3)
```

Comment:
These tests are failing, because the expected return value here should be 4.5, not 3. Lesson materials:

> Because 44 / 10 is 4.4, which we round up to the nearest 0.5 to equal 4.5.